wizard/requirements.txt

```diff
@@ -4,3 +4,4 @@ pandas
 pytest
 requests
 streamlit
+mlcroissant
```

Comment:
Alphabetical order. (Allows to easily locate packages while the requirements grow.)